packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -756,24 +792,21 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         channel.handle.attachGraph();
```

Comment:
check on line 788 seems like should be assert, as the only caller (bindChannel()) checks that this channel is in this.notBoundedChannelContextSet - another example (I think) of same state duplicated (see my comment on removal of this.notBoundedChannelContextSet and other sets in favor of asking channels these questions).